1-exercises/A-this/exercise-2.js

```diff
@@ -11,7 +11,9 @@ let person = {
     name: "Maggie",
     age: 26,
     currentAddress: "London",
-    changeAddress: /* todo */
+    changeAddress: function(newAddress){
```

Comment:
@Farnooshmo 

This article Method definitions  explains the shorthand syntax
[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Method_definitions](url)